metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -238,21 +238,19 @@ class MetalsLanguageServer(
     scribe.info(
       s"started: Metals version ${BuildInfo.metalsVersion} in workspace '$workspace'"
     )
-    clientExperimentalCapabilities =
-      ClientExperimentalCapabilities.from(params.getCapabilities)
-    initializationOptions = InitializationOptions.from(params)
 
-    languageClient.configure(clientExperimentalCapabilities)
-    languageClient.configure(initializationOptions)
+    languageClient.configure(
```

Comment:
Maybe we can just do:
```
 clientConfig.clientExperimentalCapabilities= ClientExperimentalCapabilities.from(params.getCapabilities)
 clientConfig.initializationOptions = InitializationOptions.from(params)
```

we will not need the additional configure methods.

Reply:
Ahhh, that's true.